README.md

```diff
@@ -37,21 +36,97 @@ Deployment of NVA consists of 2 templates which is used to create resources in 2
+
+ 1. Identify repositories with changes.
+ 2. Merge develop into master for each repository with changes.
+ 3. Create new Github release in repository.
+ 4. Verify CodePipeline for publishing to SAR (Serverless Repo) has run successfully by checking versions and and date in SAR.
+ 5. Update deployment templates with new applications, versions and paramters (if any), see above.
+ 6. Verify that any external resources have been updated (DynamoDB, Secrets, Elasticsearch, Cognito etc).
+ 7. Deploy templates by updating stacks in Cloudformation (from local file but ensure that branch is up to date).
+ 8. Verify changes to stack is Completed.
+ 9. Deployment is done.
+
+## NVA Applications Overview
+
+Overview of Serverless Applications in the NVA platform.
+
+### Frontend
+
+**[Frontend](https://github.com/BIBSYSDEV/NVA-frontend)**
+
+Frontend for the NVA application.
+
+### APIs
+
+These applications use DynamoDB tables. Ensure all tables and indices are up to date before updating
+application itself.
+
+**[Publication API](https://github.com/BIBSYSDEV/nva-publication-api)**
+
+For managing Publications.
+
+**[Customers API](https://github.com/BIBSYSDEV/nva-customer-api)**
+
+For managing Customer Organizations.
+
+**[DOI Requests API](https://github.com/BIBSYSDEV/nva-doi-requests-api)**
+
+For executing DOI Requests and metadata updates to DOI provider API (Datacite).
+
+**[Users and Roles Service](https://github.com/BIBSYSDEV/nva-user-access-service)**
+
+For managing users of NVA and their roles. Uses API Key stored as Secret in AWS to protect parts of 
+the API.
+
+**[Upload Multipart](https://github.com/BIBSYSDEV/nva-upload-multipart)**
+
+For uploading files associated with Publications.
+
+**[Download Publication File API](https://github.com/BIBSYSDEV/nva-download-file)**
+
+For downloading files associated with Publications.
+
+### Triggers
+
+**[Cognito Post Authentication Trigger](https://github.com/BIBSYSDEV/nva-cognito-post-authentication-trigger)**
+
+For user verification and creation. Looks up existing users or creates a default user for new logins from Customer Organizations. Uses API Key 
+stored as Secret in AWS to communicate with Users and Role Service.
+
+### Proxies to external services
+
+**[Fetch DOI](https://github.com/BIBSYSDEV/nva-fetch-doi)**
+
+Lookup, transformation and persistence of data provided by external DOI metadata services (Datacite, Crossref).
+
+**[SRU Last Publication](https://github.com/BIBSYSDEV/nva-alma-proxy)**
+
+Lookup most recent publication in library system based on name string, returns Authority identifier,
+ title and date. Accessed during first login to associate User with existing Identities in ARP.
+
+**[Person Data](https://github.com/BIBSYSDEV/nva-bare-proxy)**
+
+For looking up metadata for Person from ARP (Authority Register for Personas).
+
+**[Publication Channel Register](https://github.com/BIBSYSDEV/nva-channel-registry)**
+
+For looking up Publication Channels in NSD DBH.
```

Comment:
Please, I have no idea what NSD or DBH is. 

When using abbreviations, they _should_ be spelled out without abbreviation the first time.  This goes for every document TBH. 

What is the full names for these @brinxmat

Reply:
Had to look it up — Norsk senter for forskningsdata, Database for statistikk om høgre utdanning. Maybe better to link to https://dbh.nsd.uib.no/tjenester.action

Reply:
It also occurs to me that the naming and functionality of "Publication channel registry" application are not in sync. The application functions in practice as a proxy for all services at api.nsd.no, although we only use it for publication channel data.